src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -26,6 +38,21 @@ public class CommandPat
         private const string OutputOption = "--output";
         private const string OutputHelp = "How PAT information is displayed. [default: token]\n[possible values: none, status, token, base64, header, headervalue, json]";
 
+        private const string DomainOption = "--domain";
+        private const string DomainHelp = "The preferred domain used when acquiring Azure Active Directory access tokens. [default: microsoft.com]";
+
+        private const string AccessTokenPrompt = "AzureAuth ADO PAT";

```

Comment:
That's a great point.
1. I forgot we already add an `AzureAuth` prefix, so I'll drop that.
2. In addition to creation we also do caching and regeneration, so it's not _just_ creation...

How about `Azure DevOps PAT CLI`?

Reply:
I like that.

Reply:
Thinking more: I think `CLI` could be dropped.

Reply:
It could, but that makes it _awfully_ difficult to know what this is really about.

> `AzureAuth: Azure DevOps PAT` is requesting your credentials.
> Say what now?

vs.

> `AzureAuth: Azure DevOps PAT CLI` is requesting your credentials.
> Oh, OK, Some CLI for working with PATs wants me to auth. Sure.

Reply:
We could also just add a required `--prompt` flag and let the user be responsible for telling us what's using it.

Reply:
`AzureAuth: Azure DevOps PAT CLI` can still be a default value.

Reply:
I'm leaning towards a required `--prompt` option for 2 reasons:
1. This is how we denote usage from different tools in telemetry.
2. people generally may not even know AzureAuth is being used or what it is. What they will know is the tool they ran is creating a PAT, so the consuming tool should define the prompt. AzureAuth-Lib already has the prompt as a required argument to the token method.

Reply:
I've added a `--prompt-hint` flag (so named because it's consistent with `azureauth aad`) and I've also added a private method to validate required falgs.